packages/frontend/src/hooks/styles/useTableStyles.tsx

```diff
@@ -0,0 +1,39 @@
+import { createStyles, CSSObject } from '@mantine/core';
+
+export const useTableStyles = (overrideStyles?: Record<string, CSSObject>) =>
+    createStyles((theme) => {
+        const defaultStyles: Record<string, CSSObject> = {
+            '& thead tr': {
+                backgroundColor: theme.colors.gray[0],
+            },
+
+            '& thead tr th': {
+                color: theme.colors.gray[6],
+                fontWeight: 600,
+            },
+
+            '& thead tr th, & tbody tr td': {
+                padding: '12px 20px',
+            },
+
+            '&[data-hover] tbody tr': theme.fn.hover({
+                cursor: 'pointer',
+                backgroundColor: theme.fn.rgba(theme.colors.gray[0], 0.5),
+            }),
+        };
+
+        if (overrideStyles) {
```

Comment:
if you keep adding classes you don't need to override styles anymore. either adding classes or props (prop example in Page component)

Reply:
I went with the approach of adding the classes to the hook, let me know what you think :)